### ISSUE TITLE
Help me about using pipes

### ISSUE THREAD

**Issue by Cavid2006** [koalaman/shellcheck#2271]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2148
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
sort -t: -n -k3 ls /etc tail -15 > resultsco.txt
```

#### Here's what shellcheck currently says:
```
Line 1	SC2148: Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output

**Comment by TinCanTech** [comment]
`shellcheck` does not know which `shell` you are using. eg. **On the first-line** `#!bin/bash`
